modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -435,10 +443,74 @@ private void registerMustacheLambdas() {
         additionalProperties.put("lambda", lambdas);
     }
 
+    @SuppressWarnings({"static-method"})
```

Comment:
can you please add docstrings and more comment inside the functions to these 3 new functions?